.metalinter.json

```diff
@@ -8,8 +8,8 @@
       "Command": "gosimple -tags integration",
       "Pattern": "PATH:LINE:COL:MESSAGE"
     },
-    "maptime": {
-      "Command": "maptime -tags integration",
+    "badtime": {
+      "Command": "badtime -tags integration",
```

Comment:
could you put all the build tags we use in m3dbnode (`-tags integration,adhoc` per https://github.com/m3db/m3db/pull/399/files/c3dad4b4f019058fb6a02bbb68e458395f5b9616#diff-deadcda208014667c463990fd3af6db1)

Reply:
Sure thing.